src/panels/lovelace/special-rows/hui-attribute-row.ts

```diff
@@ -71,7 +72,8 @@ class HuiAttributeRow extends LitElement implements LovelaceRow {
                 .ts=${date}
                 .format=${this._config.format}
               ></hui-timestamp-display>`
-            : attribute ?? "-"}
+            : // will return the original attribute value if NaN
+              formatNumber(attribute, this.hass.locale) ?? "-"}
```

Comment:
We can actually check if an attribute is a number, as online states that can only be strings, attributes can be any type